www/js/components/point-map.js

```diff
@@ -169,24 +155,25 @@ class PointMap extends Component {
       );
     } else {
       view = (
-        <Map
-          center={this.state.startCenter}
-          zoom={this.state.zoom}
-          onLeafletMove={onLeafletMove}
-          onLeafletMoveEnd={ leafletMap => {
-            this.onMapMoved(leafletMap);
-          }}
-          onclick={() => {
-            dispatch(deselectMarker());
-          }}>
+        <Map  center={this.state.startCenter}
+              zoom={this.state.zoom}
+              onLeafletMove={this.props.onLeafletMove}
+              onLeafletMoveEnd={(leafletMap)=>{
+                this.onMapMoved(leafletMap);
```

Comment:
```
import { bindAll } from 'underscore'
constructor() {
  bindAll(this, 'onMapMoved');
}
```
Then, just say `onLeafletMoveEnd={this.onMapMoved}`. The `leafletMap` argument will be passed correctly.